src/components/Faq/Faq.vue

```diff
@@ -0,0 +1,339 @@
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "What is the gas fee for?",
+          ans: "Gas is the fee that is required to run computational resources to successfully execute a transaction on the Ethereum blockchain. ",
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "What is blockchain and why is it important to cryptocurrency?",
+          ans: "A blockchain refers to an immutable time-stamped series of data records supervised by a cluster of computers not owned by any single entity. Blockchain serves as the base technology for cryptocurrencies, enabling peer-to-peer transactions that are open, secure and fast.",
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "What are the application for buying DEV token?",
+          ans: "The easiest way to buy DEV tokens is to get Ethereum in a wallet and swap it for DEV on Uniswap V2",
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "Where can I use my DEV token?",
+          ans: "The DEV Tokens can be used to stake (fund) for an OSS Project, provide liquidity or used as a currency.",
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "Will DEV Token prices fluctuate?",
+          ans: "Yes. The price of the DEV token is determined by the market.",
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "What is a blockchain wallet?",
+          ans: "A blockchain wallet allows users to store and manage their Bitcoin, Ether, and other cryptocurrencies,  it also allows transactions to occur and interacts with dApps, like stakes.social.",
+          lang: "English",
+          tag: "beginners",
+        },
+        {
+          ques: "What is the mission and vision of Dev Protocol?",
+          ans: "The mission of Dev Protocol is to maximize creativity for every creator of the world, using the power of Blockchain and DeFi to redirect individuals self-interest to sustianably fund projects and creators.",
+          lang: "English",
+          tag: "developers",
+        },
+        {
+          ques: "What can you do with a Dapp, for example?",
+          ans: "You can interact with the smart contract easily with a web experience, it can be anything, from taking a loan to buying an NFT, to funding a creator, the possiblities are endless.",
+          lang: "English",
+          tag: "developers",
+        },
+        {
+          ques: "Can I establish a new market?",
+          ans: "To establish a new market, get in touch with the Dev Protocol team, preferrably through Discord, after discussions, if the market is approved it will go through the DAO voting to be implemented. ",
```

Comment:
```suggestion
          ans: "To establish a new market, get in touch with the Dev Protocol team, preferably through Discord. After discussions, if the market is approved it will go through the DAO voting to be implemented. ",
```